auth/token.go

```diff
@@ -256,12 +259,7 @@ func expired() bool {
 }
 
 func updateToken(token string, expires time.Time, filename string) {
-	select {
-	case <-WaitForDelegateKeys(nil):
-	case <-time.After(1 * time.Second):
-		log.WithField("timeout", "1s").Error("No delegate keys were available to parse the token within the timeout")
-		return
-	}
+	<-WaitForDelegateKeys(nil)
```

Comment:
Because we're waiting for a non-empty token in the controller function, there's no need to break the function of awaiting the delegate's key by using time.After